src/main/java/nl/beehivejava/scriptj/load/FilePluginLoader.java

```diff
@@ -0,0 +1,21 @@
+package nl.beehivejava.scriptj.load;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * @author Lesley
+ */
+public final class FilePluginLoader implements PluginLoader {
+
+    @Override
+    public InputStream load(String location) throws IOException {
+        Path path = Paths.get(location);
+        InputStream in = Files.newInputStream(path);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "in". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1488)